internal/otel/collector.go

```diff
@@ -31,6 +32,17 @@ type CollectorCfg struct {
 	Client            hcp.TelemetryClient
 	ForwarderEndpoint string
 	ExporterConfig    *config.ExporterConfig
+	MetricsPort       int
+	EnvoyPort         int
+	BatchTimeout      time.Duration
+}
+
+// DefaultConfig provides a config with default values for the CollectorCfg
+func DefaultConfig() CollectorCfg {
```

Comment:
Should we use this in `internal/service/service.go` to create that base config?

Reply:
I'm actually thinking we should drop this entirely because we're hard coding (no constants) some values that should be constants